fortuna/src/casus/blocks/ForBlock.js

```diff
@@ -0,0 +1,138 @@
+//@flow strict
+
+import BoundingBox from './BoundingBox.js';
+import CasusBlock from './CasusBlock.js';
+import EmptyBlock from './EmptyBlock.js';
+import Vec from './Vec.js'
+import generateCornerPerim from './generateCornerPerim.js'
+import measureText from './measureText.js';
```

Comment:
Just getting warnings that these fields are not being used. If they'll be used down the line, then keep them, if not should remove them.

Reply:
Thanks for the catch, fixed